src/Model/Endpoint.php

```diff
@@ -304,9 +330,9 @@ public function setConnection(Connection $connection)
     /**
      * Returns the connection driver.
      *
-     * @return \Muffin\Webservice\Datasource\Connection
+     * @return \Muffin\Webservice\Datasource\Connection|null
      */
-    public function getConnection(): Connection
+    public function getConnection(): ?Connection
```

Comment:
This return shouldn't be nullable, throw an exception if the property is `null`.